lib/ui/views/settings/settingsFragment/settings_manage_sources.dart

```diff
@@ -27,6 +27,7 @@ class SManageSources extends BaseViewModel {
     return showDialog(
       context: context,
       builder: (context) => AlertDialog(
+        scrollable: true,
```

Comment:
I noticed that the previous AlertDialog title was fixed in-place and the modified one is scrollable along the content with this line, as the [`title` and `content` are wrapped with the `scrollable`](https://api.flutter.dev/flutter/material/AlertDialog-class.html#:~:text=Determines%20whether%20the%20title%20and%20content%20widgets%20are%20wrapped%20in%20a%20scrollable). 

It's not that big of an issue, it looks natural (even only noticed after attaching the docs & refs) but it's the cleanest way to fully address this issue on all devices. 

<details>
<summary> 🔨 Break the glass in case of concern: </summary>

> If this is unwanted, the 'hacky' alternatives would be to add extra padding or size with a `SizedBox` or `Padding` having a magic number that could not work on some specific `MediaQuery's`, virtual keyboards, or screen sizes.
</details>

Reply:
This will be a plus because, after #1791, the title can now expand lines (jump with newlines) when it's too large to fit instead of clipping. This would make the `[AlertDialog]` content smaller and smaller (improbable but still possible), which at some point would be very hard to scroll and interact.

However, if the title and content are scrollable, this wouldn't be the case, as the user could easily scroll and focus on the content after a multi-line title.